### ISSUE TITLE
UploadCollectionItem: edit and delete button should have unique accessible names (A11y)

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents#10784]
### Describe the bug

When `UploadCollection` is set to `Detail` type, each item in the collection has a edit button and a delete button in the end, and these buttons should have unique accessible names.
Currently the buttons have no accessible names.

### Isolated Example

https://stackblitz.com/edit/github-cpmz8rrr?file=src%2FApp.tsx

### Expected Behaviour

Edit button and delete button of each item in `UploadCollection` should have unique accessible names.

### Screenshots or Videos

<img width="1015" alt="Image" src="https://github.com/user-attachments/assets/892f8249-3c72-417a-b134-032419387040" />

### UI5 Web Components for React Version

2.5.0

### UI5 Web Components Version

2.5.1

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by TeodorTaushanov** [comment]
Hello,

the buttons derive their accessible names from their tooltips. You can verify this in the Chrome Accessibility Tree view.

<img width="287" alt="Image" src="https://github.com/user-attachments/assets/9262c369-2393-42f1-8a3d-87a6f4959509" />

Best,
Teo
